plugins/dynamix.apcupsd/include/update.apcupsd.php

```diff
@@ -13,21 +13,22 @@
 ?>
 <?
 $conf  = "/etc/apcupsd/apcupsd.conf";
-$cable = $_POST['UPSCABLE']=='custom' ? $_POST['CUSTOMUPSCABLE'] : $_POST['UPSCABLE'];
+$new   = array_replace_recursive($_POST, $default);
```

Comment:
Ronald, shouldn't the `$_POST` and `$default` parameters swap positions here, so values in `$default` are overridden by values in `$_POST`?:
`$new  = array_replace_recursive($default, $_POST);`